conf/messages

```diff
@@ -126,7 +126,7 @@ generic.governmentId.error.pattern={0} must be entered in the correct format
 generic.governmentId.not.exist=The {0} you entered does not exist
 generic.error.sortCode=Enter a valid sort code like 123456
 generic.error.exactNumbers={0} must be {1} numbers
-generic.error.lookup={0} must match one of the suggestions from the list
+generic.error.lookup=No match for "{1}". Select a value from the list.
```

Comment:
I've said it twice, I'll say it for third time:

This `{1}` is not populated on overseasAddress component. This need fixing

Reply:
done